src/components/Table.jsx

```diff
@@ -0,0 +1,376 @@
+        setUpdatedAtSort(updatedAtSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleUpdatedBySort=()=>{
+        setCurrentSort("updatedBySort")
+        setUpdatedBySort(updatedBySort == "desc" ? "asc" : "desc")
+    }
+
+    const handleStatusSort=()=>{
+        setCurrentSort("statusSort")
+        setStatusSort(statusSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleBrandSort=()=>{
+        setCurrentSort("brandSort")
+        setBrandSort(brandSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleTitleSort=()=>{
+        setCurrentSort("titleSort")
+        setTitleSort(titleSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleStyleSort=()=>{
+        setCurrentSort("styleSort")
+        setStyleSort(styleSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleRowSelectIcons=(rowData, imgPath)=>{
+        return (
+            <div className="flex justify-content-end">
+                <span>
+                {(rowData.id == showEdit && isRowSelected && isAdmin) && 
+                   <span className='bg-white flex rounded-full justify-center items-center border border-grey-30  h-[30px] w-[30px]'>  
+                     <img alt={`${imgPath} svg`} src={imgPath}  style={{ fontSize: '0.6rem', color: '#708090'}} onClick={handleEditIcon}/>
+                    </span>
+                }
+                </span>
+            </div>
+        );
+    }
+
+    const handleEditIcon=()=>{
+    // code for edit icon show go in this
+    }
+
+    const pagination=()=>{
+        return(
+           <Pagination customers={customers} preText={preText} nextText={nextText} currentPage={currentPage} setCurrentPage={setCurrentPage} className={"px-3 py-2 mr-2 leading-tight text-gray-500 bg-white"}/>
+        )
+    }
+
+    const onSelectionChange = (event) => {
+        const value = event.value;
+        setSelectedProducts(value);
+    };
+
+    const onRowClick=(e)=>{
+        // row click redirection code should go in this
+    }
+
+    const onRowSelect=(e)=>{
+        setIsRowSelected(true)
+        setShowEdit(e.data.id)
+    }
+
+    const onRowUnselect=()=>{
+        setIsRowSelected(false)
+    }
+
+    return (
+        <div className='border border-grey-30 text-sx'>
+            <DataTable value={customers?.workflows} dataKey="id" rows={100} selection={selectedProducts} filterDisplay={showFilters && "row"} onRowMouseEnter={onRowSelect} onRowMouseLeave={onRowUnselect} header={ selectedProducts.length>1 && renderHeader} footer={pagination} onSelectionChange={onSelectionChange} emptyMessage="No customers found." onRowClick={onRowClick}>
+                {isAdmin && <Column selectionMode="multiple" style={{width: "3%"}}></Column>}
+                <Column field="styleId" header={<TableHeaders headerName={"Style"} sortIcon={ArrowSort} onClick={handleStyleSort}/>} filter showFilterMenu={false} filterElement={styleRowFilterTemplate} filterPlaceholder="Search by Style" style={{width: "5%"}}/>
```

Comment:
Replace all inline styles with tailwind css.

Reply:
not addressed

Reply:
data table is not accepting tailwind classes

Reply:
prime components are not accepting Tailwind classes